.gitignore

```diff
@@ -36,6 +36,9 @@ patches/**/*.patchinfo
 /third_party/cryptography
 /third_party/macholib
 /third_party/libdmg-hfsplus/
+/third_party/opengrep/opengrep
```

Comment:
do not list all files explicitly, instead unpack the downloaded archive into a subdirectory and gitignore it. This will simplify support and version bumps.